src/Chapter1.hs

```diff
@@ -556,7 +561,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+    | isInfixOf [c] "aeiouyAEIOUY" = True
```

Comment:
Oh, I see why `Data.List` is here 😸 

This is the correct solution! But as you need to check only if the char is in the list, there is `elem` function for that. It would be a bit faster than `isInfixOf` 🙂 And also, `elem` is already in `Prelude`, so no need to import that :+1:


One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎